src/components/enketo/preview.vue

```diff
@@ -40,16 +50,41 @@ export default {
         return this.$t('disabled.processing');
       return null;
     },
-    href() {
+    enketoPath() {
       // enketoId probably doesn't need to be encoded, but there is also little
       // harm.
       const encodedId = encodeURIComponent(this.formVersion.enketoId);
       return `${enketoBasePath}/preview/${encodedId}`;
+    },
+    webFormsPath() {
+      return this.formPath(
+        this.formVersion.projectId,
+        this.formVersion.xmlFormId,
```

Comment:
On form pages, `formPath()` can determine the `xmlFormId` from the route. However, it can't do that on the project overview: it needs to be explicitly passed the `xmlFormId`.